.eslintrc

```diff
@@ -8,8 +8,10 @@
     "plugin:jest/recommended",
     "plugin:cypress/recommended"
   ],
-  "plugins": ["jsdoc"],
+  "plugins": ["jsdoc", "react-hooks"],
```

Comment:
This is the official eslint plugin from the react team: https://www.npmjs.com/package/eslint-plugin-react-hooks